package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "wagtail.org",
+  "version": "1.0.0",
+  "description": "This is the source code to [Wagtail's website](https://wagtail.org)",
+  "scripts": {
+    "//[ Linting & formatting ]//": "",
+    "lint:css": "stylelint --report-needless-disables './wagtailio/static/sass'",
+    "lint:js": "eslint --ext .js,.ts,.tsx --report-unused-disable-directives .",
+    "lint:format": "prettier --check '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml}'",
+    "lint": "npm run lint:format && npm run lint:js && npm run lint:css",
+    "format": "prettier --write '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml}'",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "//[ Build tasks ]//": "",
+    "build": "webpack --mode development --progress",
+    "build:prod": "webpack --mode production",
+    "start": "webpack --mode development --progress --watch",
+    "start:reload": "webpack-dev-server"
+  },
```

Comment:
I think this would be a good time to set up GitHub Actions on the project. Here is a sample configuration, to add at `.github/workflows/ci.yml`:

```yaml
name: CI
on:
  push:
    branches:
    - main
  pull_request:
jobs:
  test_js:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
      - id: node-cache
        uses: actions/cache@v3
        with:
          path: node_modules
          key: ${{ runner.os }}-node-${{ hashFiles('**/.nvmrc') }}-${{ hashFiles('**/package-lock.json') }}
      - if: steps.node-cache.outputs.cache-hit != 'true'
        run: npm ci --no-audit
      - run: npm run lint
      - run: npm run build:prod
```

Reply:
Thanks! I've added this in and it looks to be working: https://github.com/chris-lawton/wagtail.org/actions/runs/2718091277